pulpcore/cli/file/repository.py

```diff
@@ -32,6 +33,15 @@
 _ = gettext.gettext
 
 
+def _repository_callback(
+    ctx: click.Context, param: click.Parameter, value: Optional[str]
+) -> Optional[Union[str, PulpEntityContext]]:
```

Comment:
Seems like it's simpler to use `Union[str, PulpEntityContext, None]` here. Also, I'm not sure how this would return a `PulpEntityContext`?

Reply:
right!